database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -12,7 +12,8 @@ public function up(): void
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('firstName');
+            $table->string('lastName');
```

Comment:
please use snake case for the column names to keep them consistent

Reply:
fixed